src/rust/ensogl/lib/list-view/src/entry.rs

```diff
@@ -70,6 +70,12 @@ impl Model {
     }
 }
 
+impl<T:Display> From<T> for Model {
+    fn from(item: T) -> Self {
+        Model::new(format!("{}", item))
```

Comment:
```suggestion
        Model::new(item.to_string()))
```
`ToString` trait is implemented for everything implementing Display.